src/main/java/at/hannibal2/skyhanni/data/MayorAPI.kt

```diff
@@ -195,20 +194,8 @@ object MayorAPI {
         nextMayorTimestamp = calculateNextMayorTime()
     }
 
-    private fun checkCurrentMayor() {
-        val nextMayorTime = calculateNextMayorTime()
-
-        // Check if it is still the mayor from the old SkyBlock year
-        currentMayor = candidates[nextMayorTime.toSkyBlockTime().year - 1]?.let {
-            if (it.name == lastMayor?.name) return
-
-            // TODO: Once Jerry is active, add the sub mayor perks in here
```

Comment:
you lost the todo with those changes.

Reply:
The todo actually got completed some prs ago using the Calendar menu